docs/source/reference/glossary.rst

```diff
@@ -5,3 +5,68 @@ workflow
 --------
 
 A pipeline that processes data in some way.
+
+API
+---
+
+application programming interface
+
+CLI
+---
+
+command line interface
+
+CSF/CSF3
+--------
+
+University of Manchester's Computational Shared Facility
+
+cluster
+-------
+
+see HPC
+
+DAMASK
+------
+
+Fortran package for multi-physics crystal plasticity simulation. Official documentation and installation [on their webpage](https://damask-multiphysics.org/)
+
+DAMAKS-parse
```

Comment:
typo: DAMASK

Reply:
I make a lot of typos, thanks for catching!